reflex-algos/components/Tensorflow/hello-world/main.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function
+
+import argparse
+
+
+def parse_args():
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("--arg1", help="Test argument 1")
+    options = parser.parse_args()
+    return options
+
+
+def main():
+
+    options = parse_args()
+    print("- inside hello-world.main.py Running main.py")
+    print("arg1: {}".format(options.arg1))
```

Comment:
is there a specific reason why all hello-world programs do subtly different things? if not, let try to share the code to the extent possible?

also, if this is a tf hello-world, shouldnt it confirm that infact it can do something tf related before saying things are fine?

Reply:
They should really be doing the same thing. Note that soon "Tensorflow" will be renamed "Process" (working on this in my branch), so it shouldn't actually do anything TF related.